docs/topics/mta-rn-new-features-6-2-0.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: CONCEPT
+[id="rn-new-features-6-2-0_{context}"]
+= New features
+
+This section describes the new features of the Migration Toolkit for Applications (MTA) 6.2.0.
+
+.Integration with JIRA
+A new _Jira_ entry has been added in the left sidebar, under the Control Menu. This feature enables the user (Administrator, Architect) to create new Jira instances and edit existing instances.
```

Comment:
I do not think this is a good description of the feature. I have an issue, pedantic albeit, with lists in brackets as I do not think it is good grammar. 

@sbeskin-redhat - please could you help with the wording here?

Reply:
Yes, I am reviewing these release notes.

Reply:
@HagayVider1 
I would suggest this description of this feature:
"Integration with Jira enables users to track migrations of individual applications by creating a Jira issue for each migration from within the {ProductShortName} {WebName}."